examples/liquid.ini

```diff
@@ -149,6 +149,7 @@ two_factor_auth = false
 ;hoover_ui_async_search_error_multiplier = 2
 ;hoover_ui_async_search_error_summation = 60
 
+
```

Comment:
extra whitespace

Reply:
I don't get it in my source code so don't know where that come from, either way I resolve, we can remove that later